deployment/v3/docs/external services configuration.md

```diff
@@ -0,0 +1,70 @@
+#  External Services Configuration
+
+* This document describes how to configure external services like minio,keycloak and postgres
+
+
+## postgres external configuration procedure
+
+* Go to mosip-config and checkout to specific branch to update config properties.
+* Default value for database hostname is `postgres-postgresql.postgres` and  Default value for database port number is `5432`.
+* If you are using external service, then you have to update hostname and port number with this below sed command and provide the new hostname
+  ```
+   cd mosip-config
+   sed -i 's/postgres-postgresql.postgres/new-hostname/g' *
+   sed -i 's/5432/new-port number/g' *
```

Comment:
please use `<postgres-external-hostname>` and `<postgres-external-port>` instead of new-hostname and new-port number.

```
   sed -i 's/postgres-postgresql.postgres/<postgres-external-hostname>/g' *
   sed -i 's/5432/<postgres-external-port>/g' *
```